src/View/Helper/SearchFormHelper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Search\View\Helper;
+
+use Cake\ORM\TableRegistry;
+use Cake\View\Helper;
+use Cake\View\View;
+
+/**
+ * SearchForm helper
+ */
+class SearchFormHelper extends Helper
+{
+
+    public $helpers = ['Form'];
+
+    /**
+     * Default configuration.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [];
+
+    /**
+     * Output a complete search form, based on the search configurations set up
+     * in the specified table
+     *
+     * @param string $model The Table to load
+     * @return null|string An formatted FORM with inputs.
+     */
+    public function form($model)
+    {
+        $table = TableRegistry::get($model);
+
+        // make sure the table has the behavior and has implemented the searchConfiguration method
+        if (! $table->behaviors()->has('Search') ||
```

Comment:
Interesting that it passed the Travis / PHP-CS as-is... but I'll make the change.